From d9804182eae43cf49f735e6533ddbe1541c2b9a9 Mon Sep 17 00:00:00 2001
From: jamesread <contact@jread.com>
Date: Mon, 2 Mar 2026 00:29:02 +0000
Subject: [PATCH] security: GHSA-4fqm-6fmh-82mq Authentication bypass in
 KillAction - thanks for the responsible disclosure @kule500 and making
 OliveTin better

---
 service/internal/config/sanitize.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/service/internal/config/sanitize.go b/service/internal/config/sanitize.go
index f699b803..ddeac5b9 100644
--- a/service/internal/config/sanitize.go
+++ b/service/internal/config/sanitize.go
@@ -164,6 +164,7 @@ func (cfg *Config) sanitizeAuthRequireGuestsToLogin() {
 		cfg.DefaultPermissions.View = false
 		cfg.DefaultPermissions.Exec = false
 		cfg.DefaultPermissions.Logs = false
+		cfg.DefaultPermissions.Kill = false
 	}
 }
 
